scripts/DeleteTestOutputs.ps1

```diff
@@ -0,0 +1,32 @@
+<#
+.SYNOPSIS
+    Delete test outputs from bld\bin to reduce published content size.
```

Comment:
>Delete [](start = 4, length = 6)

I don't see anybody calling this. Is it intended as a separate step in a build pipeline definition?

Reply:
Yes. Our new Azure DevOps build calls this near the end. Developers don't need to call this for local builds, so I didn't want to tie it in anywhere by default.